### PR TITLE
Implements toggle button

### PR DESCRIPTION
**Pull request by t1mato**
Created a toggle button that shows whether or not the extension is enabled. The button logs whether it is activated or not through true (on) and false (off).

### CODE REVIEW

**Review by Leaversa — CHANGES_REQUESTED** [review]
I believe you need to add in linting and I'm not sure if the button is being logged properly.

**Review by Leaversa — APPROVED** [review]

File: src/ToggleButton.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Box, FormControlLabel, Switch } from '@mui/material';
+import { useState } from 'react';
```
Comment (GuyWhoCode) [inline src/ToggleButton.tsx:3]: Is there a particular reason why you explicitly exported `useState` from React if you already imported everything? Does it not let you use that variable if you imported everything from React in line 1?

**Review by GuyWhoCode — CHANGES_REQUESTED** [review]

File: src/ToggleButton.tsx (hunk 1 above)
Comment (t1mato) [reply src/ToggleButton.tsx:3]: Yeah, when I delete useState it says that it can't find the name for it.

**Review by GuyWhoCode — APPROVED** [review]

**Comment by misslame** [comment]
This is ready to squash and merge after meeting 9/30

**Pull request merged**